lib/plugins/helper/open_graph.js

```diff
@@ -20,21 +20,39 @@ const localeMap = {
   'tr': 'tr_TR',
   'vi': 'vi_VN'
 };
-const localeRegex = new RegExp(Object.keys(localeMap).join('|'), 'i');
+const localeCache = new Cache();
+const localeToTerritory = str => localeCache.apply(str, () => {
+  if (str.length === 2 && localeMap[str]) return localeMap[str];
+
+  if (str.length === 5) {
+    let territory = [];
+    if (str.includes('-')) {
+      territory = str.split('-');
+    } else if (str.split('_')) {
```

Comment:
can be just `else` since `str.split('_')` will never return falsy value.

Reply:
LGTM then. I will update to that.